docs/MAPS.md

```diff
@@ -1,7 +1,7 @@
 # MWM Files
 
 MAPS.ME uses maps in its own vector format, MWM. It contains classified features sorted and simplified by zoom level.
-For car routing, it needs a separate routing index in a `.mwm.routing` file. We build maps for the entire planet:
+We build maps for the entire planet:
```

Comment:
I think the removed part should be replaced with "It also can include a pre-calculated routing index for car routing."